mlrun/api/db/sqldb/db.py

```diff
@@ -979,3 +1186,16 @@ def _add_utc_timezone(obj, attribute_name):
         https://stackoverflow.com/questions/6991457/sqlalchemy-losing-timezone-information-with-sqlite
         """
         setattr(obj, attribute_name, pytz.utc.localize(getattr(obj, attribute_name)))
+
+    @staticmethod
+    def _transform_feature_set_model_to_scheme(db_fs: FeatureSet) -> schemas.FeatureSet:
```

Comment:
```suggestion
    def _transform_feature_set_model_to_schema(db_fs: FeatureSet) -> schemas.FeatureSet:
```
I have this typo too, will fix it

Reply:
done